contents/customers/carvertical.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How carVertical uses feature flags and analytics without autocapture
+customer: carVertical
+logo: ../images/customers/carvertical/carvertical_logo.png
+logoDark: ../images/customers/carvertical/carvertical_logo_dark.png
+featuredImage: ../images/customers/carvertical/featured.jpg
+industries:
+    - Recruitment
+users:
+    - Growth
+    - Engineering
+    - Product
+toolsUsed:
+    - Feature flags
+    - Product analytics
+date: 2023-12-01
+---
+
+[carVertical](https://www.carvertical.com/) is the world’s leading automotive data company that builds vehicle history reports from accumulated data. If you’re looking to buy a used car these reports can help you understand the specific history of a particular vehicle, as well as help you to understand more about the overall model.
+
+As befits a company which processes mountains of data on a regular basis, carVertical’s product and growth teams are incredibly data-led. In the past, they’ve used tools such as Google Analytics to help them understand user behavior, but as the company has grown, they’ve needed to get more sophisticated.
```

Comment:
Just a random observation, but would be cool if we could add some context to how much data they process. Would  underline that we can handle that scale.